db/schema.rb

```diff
@@ -73,7 +80,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "disabled"
-    t.bigint "building_id"
+    t.bigint "building_id", default: 1
```

Comment:
Does it work to just have this added to the schema.rb, or do we need to add a migration to add it?

Reply:
Hmmmm... this doesn't even seem related to the original rubocop complaint.  I'm removing it.